homeassistant/components/sensor/vera.py

```diff
@@ -146,11 +146,12 @@ def state_attributes(self):
 
         if self.vera_device.is_trippable:
             last_tripped = self.vera_device.refresh_value('LastTrip')
-            trip_time_str = time.strftime(
-                "%Y-%m-%d %H:%M",
-                time.localtime(int(last_tripped))
-            )
-            attr[ATTR_LAST_TRIP_TIME] = trip_time_str
+            if last_tripped is not None:
+                utc_time = dt_util.utc_from_timestamp(int(last_tripped))
+                attr[ATTR_LAST_TRIP_TIME] = dt_util.datetime_to_local_str(
```

Comment:
At the moment the only last trip time gets displayed when the more-info-default gets rendered, I can remove the conversion to local time but then the date displayed on the more-info card is misleading, It's easy to change so I will update to not be local.  Maybe a way of converting dates to the correct timezone in the more-info-default card should be implemented when it is looping through attributes to display them on the UI.